content/influxdb/v2.1/notebooks/troubleshoot-notebooks.md

```diff
@@ -10,10 +10,13 @@ menu:
 ---
 
 ### No measurements appear in my bucket even though there's data in it.
-Try changing the time range. You might have measurements prior to the time range you selected. For example, if the selected time range is `Past 1h` and the last write happened 16 hours ago, you'd need to change the time range to `Past 24h` (or more) to see your data.
+
+Try changing the time range. You might have measurements prior to the time range you  selected. For example, if the selected time range is `Past 1h` and the last write happened 16 hours ago, you'd need to change the time range to `Past 24h` (or more) to see your data.
 
 ### "No bucket exists" error message appears.
-This error appears when the Buckets API endpoint returns a list of bucket and you don’t have access to that bucket. Verify that you have access permissions to the bucket that you're trying to query or write to.
 
-### Metric selector crashes.
+This error occurs when the [Buckets API endpoint](/influxdb/v2.1/api/#tag/Buckets) returns a list of bucket and you don’t have access to that bucket. Verify that you have access permissions to the bucket that you're trying to query or write to. 
+
+### My notebook crashes or is running slowly.
+
```

Comment:
Do we want to remove this section altogether now that metric selector has been removed? Or do we think there may be cases something crashes, and checking/resolving cardinality would fix?

Reply:
Yes, I would remove this section. The query builder is basically the same for data explorer and for notebooks, so we don't need to special-case performance here.